packages/core-svelte/src/lib/components/TextComponent.svelte

```diff
@@ -688,9 +687,9 @@
     <ErrorMarker {editor} {box} />
 {/if}
 <ErrorTooltip {editor} {box} {hasErr} parentTop={0} parentLeft={0}>
-    <span {id} role="none" bind:this={surroundingElement}>
+    <span {id} role="none" bind:this={surroundingElement} class="text-component-parent">
```

Comment:
Prefer to call the CSS class "text-component", as this is in line with the names we use in other components.